lib/pa_ess/http_updater.ex

```diff
@@ -141,7 +149,15 @@ defmodule PaEss.HttpUpdater do
           |> URI.encode_query()
 
         {arinc_ms, signs_ui_ms, result} = send_payload(encoded, state)
-        log("send_audio", encoded, arinc_ms: arinc_ms, signs_ui_ms: signs_ui_ms, sign_id: sign_id)
+
+        log("send_audio", encoded,
+          arinc_ms: arinc_ms,
+          signs_ui_ms: signs_ui_ms,
+          sign_id: sign_id,
+          message_type: to_string(struct) |> String.split(".") |> List.last(),
+          message_details: Map.from_struct(audio) |> inspect()
```

Comment:
Thinking out loud: Logging the whole struct is probably ok for now, but we should keep in mind that the refactoring effort is aiming to substantially change or remove many of these structs, so we shouldn't build any dashboards or alerting based on their internal contents.